examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/PurchaseOptionExtensions.kt

```diff
@@ -0,0 +1,10 @@
+package com.revenuecat.purchasetester
+
+import com.revenuecat.purchases.models.PurchaseOption
+
+fun PurchaseOption.toButtonString(): String {
+    val pricingPhasesString = pricingPhases.joinToString(separator = ",\n") { pricingPhase ->
+        "\t[${pricingPhase.formattedPrice}, ${pricingPhase.billingPeriod}, ${pricingPhase.billingCycleCount} cycles]"
+    }
+    return "PricingPhases = [\n$pricingPhasesString\n],\nTags = $tags"
```

Comment:
Honestly, this is very crowded and ugly in the UI... But purchase tester being an internal test app, I thought it was ok with it being dirty for now. We can maybe move choosing between the purchase options to a different screen in the future, but I thought it would be useful to have all the package information in the same place.